.github/workflows/lambda-functions.yml

```diff
@@ -85,7 +85,14 @@ jobs:
 
       - name: Package and deploy lambda function
         run: |
-          zip -ug vetext_incoming_forwarder_lambda.zip vetext_incoming_forwarder_lambda.py
+          python3 -m venv venv
+          source venv/bin/activate
+          pip install -r requirements-lambda.txt
+          deactivate
+          cd venv/lib/python3.8/site-packages
+          zip -r9 ../../../../vetext_incoming_forwarder_lambda.zip .
+          cd ../../../../
+          zip -ugj vetext_incoming_forwarder_lambda vetext_incoming_forwarder_lambda.py
```

Comment:
Is there a reason not to roll this into "deploy-other-lambdas" now?  "role-duration-seconds" seems to be the only difference.

Reply:
Yes, could probably roll this into other lambdas.  I'd argue that it's probably better to have all lambdas be on their own so we could deploy a single lambda without deploying all the rest, probably by having another input asking for which lambda to deploy.  If that field is empty then it can deploy all of them.  Long story short, I'll probably keep this as is for now.